app/src/androidTest/java/ch/epfl/balelecbud/SocialActivityTest.java

```diff
@@ -81,10 +82,10 @@ private void createFriendship(User user) {
         mockDb.storeDocumentWithID(DatabaseWrapper.FRIENDSHIPS_PATH, user.getUid(), toStore);
     }
 
-    private void createRequestFromUser(User user) {
+    private void createRequest(User from, User to) {
```

Comment:
You could replace `FriendshipUtils.addFriend(User)` with this function as they are similar